src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts

```diff
@@ -160,6 +162,8 @@ actionRegistry.registerWorkbenchAction(SyncActionDescriptor.from(DisableAutoUpda
 actionRegistry.registerWorkbenchAction(SyncActionDescriptor.from(InstallSpecificVersionOfExtensionAction), 'Install Specific Version of Extension...', ExtensionsLabel);
 actionRegistry.registerWorkbenchAction(SyncActionDescriptor.from(ReinstallAction), 'Reinstall Extension...', localize({ key: 'developer', comment: ['A developer on Code itself or someone diagnosing issues in Code'] }, "Developer"));
 
+actionRegistry.registerWorkbenchAction(SyncActionDescriptor.from(OpenExtensionAuthoringDocsAction), 'Extensions: Author an Extension...', ExtensionsLabel);
```

Comment:
Put this in https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/contrib/extensions/browser/extensions.contribution.ts#L1